PMM-7278 Updated MongoDB TLS doc for auth mechanism (#571)
percona/pmm-doc: docs/setting-up/client/mongodb.md

--tls-certificate-key-file=PATHTOCER \
--tls-certificate-key-file-password=IFPASSWORDTOCERTISSET \
--tls-ca-file=PATHTOCACERT
--authentication-mechanism=AUTHENTICATION-MECHANISM
--authentication-database=AUTHENTICATION-DATABASE
```

where:

- `PATHTOCERT`: Path to TLS certificate file.
- `IFPASSWORDTOCERTISSET`: Password for TLS certificate file.
- `PATHTOCACERT`: Path to certificate authority file.

> - `AUTHENTICATION-MECHANISM`: Authentication mechanism. Default is empty. Use MONGODB-X509 for ssl certificates.

Reviewer: ```suggestion
- `AUTHENTICATION-MECHANISM`: Authentication mechanism. Default is empty. Use `MONGODB-X509` for SSL certificates.
```

Reviewer: @percona-csalguero are there any other possible values here? Where or how would a user find out what they should use?

Author: The default is `SCRAM` and `MONGODB-X509` is for X509 certs.
Other values depend on the server config: https://docs.mongodb.com/manual/reference/parameters/#mongodb-parameter-param.authenticationMechanisms

> - `AUTHENTICATION-DATABASE`: Authentication database. Default is empty. Use $external for ssl certificates.

Reviewer: Is `$external` a literal string or a variable expansion? Are there any issues with quoting? Can you give an example of use. Plus, as above, what other values are possible and where are they found?

Reviewer: ```suggestion
- `AUTHENTICATION-DATABASE`: Authentication database. Default is empty. Use `$external` for SSL certificates.
```

2. Look in the *Services* tab for a matching *Service Type* (MongoDB), *Service name*, *Addresses*, and any other values used when adding the service.
3. Look in the *Agents* tab to check the desired data source is being used.
4. If your MongoDB instance is configured to use TLS, click on the **Use TLS for database connection** check box and fill in TLS certificates and keys.

>     1. If you use TLS, the authentication mechaninsm is being set automatically to `MONGODB-X509`.

Reviewer: ```suggestion
    1. If you use TLS, the authentication mechanism is automatically set to `MONGODB-X509`.
```
